src/main/webapp/player/simid_player.html

```diff
@@ -61,6 +62,9 @@
           ];
           adParams = JSON.stringify(adParamsObj);
           break;
+        case 'maps/simid_map.html':
+          document.getElementById('geolocation').style.visibility = 'visible';
+          break;
```

Comment:
other code changes adParams here. Do you need to do that. If not consider emptying that field.

Reply:
fixed